### ISSUE TITLE
I can't use it

### ISSUE THREAD

**Issue by aazz-top** [gawindx/WinNUT-Client#182]
Exception type: System.ArgumentException
Exception message: TimeSpan 不接受浮点非数字值。
Exception stack trace: 
   在 System.TimeSpan.Interval(Double value, Int32 scale)
   在 WinNUT_Client.WinNUT.Update_UPS_Data()
   在 WinNUT_Client.WinNUT.UPS_Connect()
   在 WinNUT_Client.WinNUT.WinNUT_Load(Object sender, EventArgs e)
   在 System.EventHandler.Invoke(Object sender, EventArgs e)
   在 System.Windows.Forms.Form.OnLoad(EventArgs e)
   在 System.Windows.Forms.Form.OnCreateControl()
   在 System.Windows.Forms.Control.CreateControl(Boolean fIgnoreVisible)
   在 System.Windows.Forms.Control.CreateControl()
   在 System.Windows.Forms.Control.WmShowWindow(Message& m)
   在 System.Windows.Forms.Control.WndProc(Message& m)
   在 System.Windows.Forms.ScrollableControl.WndProc(Message& m)
   在 System.Windows.Forms.ContainerControl.WndProc(Message& m)
   在 System.Windows.Forms.Form.WmShowWindow(Message& m)
   在 System.Windows.Forms.Form.WndProc(Message& m)
   在 System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)
   在 System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   在 System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)

**Comment by gbakeman** [comment]
Hi,

This repository is no longer maintained, and development has moved [here](https://github.com/nutdotnet/WinNUT-Client).

**Comment by aazz-top** [comment]
> Hi,
> 
> This repository is no longer maintained, and development has moved [here](https://github.com/nutdotnet/WinNUT-Client).
think you, can you help me?
https://github.com/nutdotnet/WinNUT-Client/issues/129#issuecomment-1926938936

**Comment by gbakeman** [comment]
Yes, sorry - I will head over to your issue right now. I just want to make sure others seeing this report will know where to go.
